lookout/style/format/quality_report.py

```diff
@@ -36,19 +71,304 @@ def quality_report(input_pattern: str, bblfsh: str, language: str, n_files: int,
 
     y_pred, _ = rules.predict(X, vnodes_y, vnodes, language)
 
-    target_names = [CLASSES[cls_ind] for cls_ind in numpy.unique(y)]
-    print("Classification report:\n" + classification_report(y, y_pred, target_names=target_names))
-    print("Confusion matrix:\n" + str(confusion_matrix(y, y_pred)))
+    target_names = [CLASSES[cls_ind] for cls_ind in unique_labels(y, y_pred)]
+    print(generate_report(y=y, y_pred=y_pred, target_names=target_names, vnodes_y=vnodes_y,
+                          n_files=n_files))
+
+
+class ReportModel(AnalyzerModel):
+    """Model for ReportAnalyzer - store configuration."""
+
+    NAME = "style.format.quality_report_analyzer.ReportModel"
+
+    def __init__(self, config=None, **kwargs):
+        """Initialize ReportModel with configuration."""
+        super().__init__(**kwargs)
+        self.config = config
+
+    def _generate_tree(self) -> dict:
+        """
+        Return the tree to store in ASDF file.
+
+        :return: dict.
+        """
+        return dict(config=[deepcopy(self.config)])
+
+    def _load_tree(self, tree: dict) -> None:
+        """
+        Attach the needed data from the tree.
+
+        :param tree: asdf file tree.
+        :return: None
+        """
+        self.config = tree
```

Comment:
These two methods above do not agree with each other. The first places the config under "config" key in the dict.
The second assigns the whole tree to config, so that the config is always `{"config": config}`. Change either the first or the second method.